### DISCUSSION TITLE
Distinguish vendor accounts

### DISCUSSION THREAD

**Discussion by discover59** [amzn/selling-partner-api-models#3518] [General]
My client has two vendor central accounts and I get different advertising metrics for those accounts.

But when he authorized selling partner API with those accounts, I get same data for report type `GET_VENDOR_SALES_REPORT`.

I need an individual report for each vendor account, but cannot figure it out now.

Is there any option in the report request I am missing?

**Comment by MeenaAmz** [comment]
Each account will have its own LWA refresh token which you will use to get the access token. You should pass in the appropriate token to get the report for that account. Are you a third party developer and is your client authorizing your app through Oauth?

**Reply by discover59** [reply]
I am a 3rd party developer. Is there any relation between vendorcentral group id and advertising entity id?
